spec/buildCommitSpec.js

```diff
@@ -98,4 +98,61 @@ line 2`;
       expect(buildCommit(answersNoScope, options)).toEqual(expecteMessage);
     });
   });
+
+  describe('with ticketNumberSuffix', () => {
+    it('should be visible', () => {
+      const answersTicketNumberSuffix = {
+        ...answers,
+        ticketNumber: '12345',
+      };
+      const options = {
+        allowTicketNumber: true,
+        ticketNumberSuffix: '@@@ ',
+      };
+
+      expect(buildCommit(answersTicketNumberSuffix, options)).toEqual('feat(app): 12345@@@ this is a new feature');
+    });
+  });
+
+  describe('with ticketNumberPosition', () => {
+    it('should be same', () => {
+      const answersTicketNumberSuffix = {
+        ...answers,
+        ticketNumber: '12345',
+      };
+      const options = {
+        allowTicketNumber: true,
+        ticketNumberPosition: 'standard',
+      };
+
+      expect(buildCommit(answersTicketNumberSuffix, options)).toEqual('feat(app): 12345 this is a new feature');
+    });
+
+    it('should be "first"', () => {
+      const answersTicketNumberSuffix = {
+        ...answers,
+        ticketNumber: '12345',
+      };
+      const options = {
+        allowTicketNumber: true,
+        ticketNumberPosition: 'first',
+      };
+
+      expect(buildCommit(answersTicketNumberSuffix, options)).toEqual('12345 feat(app): this is a new feature');
+    });
+
+    it('should be "last"', () => {
+      const answersTicketNumberSuffix = {
+        ...answers,
+        ticketNumber: '12345',
+      };
+      const options = {
+        allowTicketNumber: true,
+        ticketNumberSuffix: '',
+        ticketNumberPosition: 'last',
+      };
+
+      expect(buildCommit(answersTicketNumberSuffix, options)).toEqual('feat(app): this is a new feature12345');
```

Comment:
does ticket number need space here?

Reply:
not sure I fully understand you here. Space character is default for _ticketNumberSuffix_, so it will be there. But it doesn't make really sense, thus implementing _ticketNumberSuffix_ option in first place, and in this test case its set to '' (empty string) on line 151, as I find it "most common use case" for setting _ticketNumberPosition_ to "last"

Reply:
I m referring to the result `'feat(app): this is a new feature12345'`. 
Do you intent to have `'feat(app): this is a new feature12345'` or `'feat(app): this is a new feature 12345'`? Note space before `12345`

Reply:
I let this to _ticketNumberPrefix_ option we already have but it will be nice to show it here. That's right.